include/aspect/parameters.h

```diff
@@ -123,7 +123,7 @@ namespace aspect
        */
       enum Kind
       {
-        boussinesq_approximation,
+        Boussinesq_approximation,
```

Comment:
I think we should keep it lower-case here, just like we have functions such as `solve_stokes()` etc.